### ISSUE TITLE
How is conditioning dropout done in training?

### ISSUE THREAD

**Issue by max-fofanov** [ShineChen1024/oms-Diffusion#106]
First of all thank you for your awesome work. I wonder how do you do conditioning dropout? If you pass torch.zeros_like(...) like you do on inference how can this be utilized when training? Passing the torch.zeros_like(...) into Unet instead of garment encoder input doesn't allow gradients to flow, as far as I understand. And if you pass torch.zeros_like as an input to garment encoder, why do you inference the model in a different way, creating torch.zeros_like directly in the attention processor instead of creating it before using ref_unet and passing it through it. Thank you for the answer!

**Comment by max-fofanov** [comment]
@ShineChen1024
